RELEASE_PROCESS.rst

```diff
@@ -7,9 +7,33 @@ Make the clone using:
 
 ::
 
-     git clone --branch master git@github.com:juju/juju-gui.git
-     cd juju-gui
-     git merge origin/develop
+    git clone git@github.com:juju/juju-gui.git gui-release
+    cd juju-gui
+
+Next you'll want to generate the list of changes for CHANGES.yaml.
+
+::
+
+    git log <last release>...HEAD --author 'jujugui' --format='* [%h] %b'
+
+Update CHANGES.yaml to include the list of changes--you can try to summarize
+the list of commits or just include them. If you do the latter, note that
+the list of commits may still need to be cleaned up to fit appropriately in the
+CHANGES file. Look at previous entries to see how it should look.
+
+Commit the changes to the changelog.
+
+::
+
+    git commit -am "Updating changelog."
```

Comment:
@hatched Yeah, see what @bac said--the supported release directions start with a fresh checkout. You should *never* cut the release from your working dir. We've had issues with that in the past.